carbon/components/Skeleton/index.tsx

```diff
@@ -1,9 +1,18 @@
 import { t } from "@lingui/macro";
 import styles from "./styles.module.scss";
-export default function Skeleton(props: { className?: string; text?: string }) {
-  const text = props.text === "undefined" ? t`Fetching data...` : props.text;
+export default function Skeleton(props: {
+  className?: string;
+  text?: string;
+  height?: number;
+}) {
+  const text = props.text === undefined ? t`Fetching data...` : props.text;
+  const style = props.height
+    ? {
+        minHeight: props.height,
+      }
+    : {};
```

Comment:
can we use a classname and proper css here instead?

Reply:
Done in 821f1568